home-tasks/src/test/resources/suites/additionAndSubtractOperations.xml

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="SimpleFunctions" verbose="1" >
+  <test name="AdditionAndSubtractTest">
+    <classes>
+      <class name="com.epam.tc.hw1.functions.SubtractOperationTestDouble"/>
+      <class name="com.epam.tc.hw1.functions.SubtractOperationTestLong"/>
+      <class name="com.epam.tc.hw1.functions.AdditionOperationTestDouble"/>
+      <class name="com.epam.tc.hw1.functions.AdditionOperationTestLong"/>
+    </classes>
```

Comment:
Try to use class level `groups` instead of explicit declarations
http://makeseleniumeasy.com/2018/07/03/testng-tutorials-30-groups-attribute-at-class-level-in-testng/